packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -244,6 +250,21 @@ const Dashboard = ({
                         </div>
                         <div icon='IcChartsTabDbot' label={localize('Charts')} id='id-charts'>
                             <Chart />
+                            {active_tab !== 2 && (
+                                <MobileWrapper>
+                                    <div className='mobile-wrapper'>
+                                        <Collapsible position='top' is_collapsed={false} as='div'>
+                                            <ExtendedDiv className='dashboard__run-strategy-wrapper' collapsible>
+                                                {active_tab !== 2 && <RunStrategy />}
```

Comment:
i think we dont require `active_tab !== 2` here!

Reply:
@rupato-deriv we don’t need RunStrategy component on the Quick Strategy tab = 2

Reply:
![Uploading Screenshot 2022-12-27 at 3.32.50 PM.png…]()

Reply:
<img width="1178" alt="Screenshot 2022-12-27 at 3 32 50 PM" src="https://user-images.githubusercontent.com/97010868/209629579-a1a61104-6ada-4506-a9ae-be8e19b62fcf.png">

Reply:
<img width="1151" alt="image" src="https://user-images.githubusercontent.com/97010868/209629685-95786857-920f-4dc5-a4b8-52bd7b1f61b9.png">
i meant this two checks we dont require inner check

Reply:
fixed